tools/twix/src/panels/look_at.rs

```diff
@@ -38,14 +36,14 @@ impl Panel for LookAtPanel {
     const NAME: &'static str = "Look At";
 
     fn new(nao: Arc<Nao>, _: Option<&Value>) -> Self {
-        let (field_dimensions_update_notify_sender, field_dimensions_update_notify_receiver) =
-            mpsc::channel(1);
+        let (field_dimensions_update_notify_sender, _) = mpsc::channel(1);
         let field_dimensions_buffer = subscribe(
             nao.clone(),
             "field_dimensions",
             field_dimensions_update_notify_sender,
         )
```

Comment:
You don't even need to create a channel here. You can simply subscribe the parameter directly.
Look at what the `subscribe` function does.

Reply:
No no no :D
You don't need a sender either. Simply calling `nao.subscribe_parameter(path)` gives you the `ValueBuffer` you want.
That's what `subscribe` does, but that also does some minor input validation, calls `nao.subscribe_parameter(path)` and subscribes to updates with the passed sender.

Reply:
Fixed, sorry for the delay.